### ISSUE TITLE
Wii Version - How do you get this working?

### ISSUE THREAD

**Issue by Cee123** [libretro/dosbox-svn#27]
I'm trying to get this core working but it just freezes on a black screen after I hit "Load Core". Not really sure that this works.

**Comment by fr500** [comment]
I haven't tested on Wii personally. There is a chance it doesn't

**Comment by Cee123** [comment]
I've tested it, tried a few different things. I tried with a keyboard plugged in thinking it might be that, but nothing happens. I've tried with a .conf in the DosBox directory on the SD card but nothing happens either. So I'm not really sure. There are no instructions and can't find anything mentioned anywhere about DosBox SVN on Wii. Not sure if it's something I'm doing wrong, or whether it just doesn't work. As it just freezes on a black screen.

**Comment by fr500** [comment]
I just know it's compiling, but I have never tested it. I tried it in dolphin and it's crashing there too so I guess it's not working.

Wouldn't really know how to debug this.

**Comment by Cee123** [comment]
Oh ok, if it's crashing in Dolphin there must be something wrong with it then.

**Comment by Cee123** [comment]
The core seems just crashes to a black screen upon loading.

**Comment by realnc** [comment]
There's nobody here who can look into this. I certainly don't have a Wii (nor do I want one.) So, if anyone else can help, that would be great.

**Comment by Cee123** [comment]
@realnc Okay, no worries.

**Comment by Cee123** [comment]
> There's nobody here who can look into this. I certainly don't have a Wii (nor do I want one.) So, if anyone else can help, that would be great.

Yeah no that's fine. I completely understand. Not that I'm bothered or even care that much about it. I was just messing with the wii version of RA out of boredom and noticed dosbox doesn't work, so thought I'd post it as an issue here anyway.
